docs/support.mdx

```diff
@@ -1,111 +1,35 @@
 ---
-title: Getting help
+title: Support
 description: How to find answers to questions or problems you encounter while using IBM Quantum (Platform, or on IBM Cloud) or Qiskit Runtime
 
 ---
 
-# Getting help
+# Support
 
-## IBM Quantum Support
+## How to get help
 
-Members of the IBM Quantum&trade; Network can reach out to IBM Quantum Support if they have technical difficulties, questions, or concerns. Ask your administrator or IBM&reg; representative for IBM Quantum Support contact information.
+| | |
+| --- | --- |
+| IBM Quantum&trade; Network members  | Reach out to IBM Quantum Support if you have technical difficulties, questions, or concerns. Ask your administrator or IBM&reg; representative for IBM Quantum Support contact information. |
+| Pay-As-You-Go users  | Open a case in the IBM Cloud&reg; Support Center by following [this link](https://cloud.ibm.com/unifiedsupport/cases/form), and choosing 'Qiskit Runtime' from the Topic field. |
+| Qiskit SDK users | Access the Qiskit Slack community: [Qiskit Slack](https://qisk.it/join-slack).  |
+| Qiskit Runtime users | Join the `qiskit-runtime` channel within the [Qiskit Slack workspace](https://qisk.it/join-slack).|
+|Anyone|Discuss quantum information science and development with the larger quantum computing field on the [Quantum Computing Stack Exchange](https://quantumcomputing.stackexchange.com/questions/) site. Be sure to read the ["How do I ask a good question?"](https://quantumcomputing.stackexchange.com/help/how-to-ask) guide, and make use of tags such as `qiskit`, `ibm-runtime`, and `ibm-quantum-services` for best results.|
+|Questions specific to programming|Visit [Stack Overflow](https://stackoverflow.com/) and use the tag `qiskit`.|
 
-Pay-As-You-Go users can open a case in the IBM Cloud&reg; Support Center by following [this link](https://cloud.ibm.com/unifiedsupport/cases/form), and choosing 'Qiskit Runtime' from the Topic field.
+## Find Qiskit on GitHub
```

Comment:
For this section I think we could also add qiskit-serverless.

Also would be good to add some intro text about the kinds of things users might get from going to the github (e.g. open bug reports and feature requests, see whats coming in future releases, make contributions etc.)

Reply:
In fact, maybe this section could be combined somehow with the open source governance section below.